extensions/wikia/AdEngine/js/wad/babDetection.js

```diff
@@ -39,7 +39,12 @@ define('ext.wikia.adEngine.babDetection', [
 		});
 	}
 
+	function isBlocking() {
+		return win.ads && win.ads.runtime && win.ads.runtime.bab && win.ads.runtime.bab.blocking;
```

Comment:
Is it possible to have some race conditions here? Is there any chance that isBlocking is going to be called before detection is finished?

Reply:
It should not. WAD is quite fast and probably will always finish detection before recovery initialisation. But we should probably test it more intense during QA.